### DISCUSSION TITLE
How to add SSL certificate?

### DISCUSSION THREAD

**Discussion by michaelliesenberg** [SVG-Edit/svgedit#707] [General]
Hi,
how do i set my private certificate and key using the command "npm run start"?

**Comment by jfhenon** [comment]
I'm not sure that the server used for development supports certificates.
The documentation is: https://modern-web.dev/docs/dev-server/overview/
You may have to switch to a "real" web server (for example Apache)

**Comment by michaelliesenberg** [comment]
Is there any example showing how to get the svgedit to run on apache?
I have set the apache server but i am getting issues one after the other.
I thought i could show apache the "svgedit/src/editor/index.html" and i would run normally, but it doesnt...

**Comment by jfhenon** [comment]
It will not work with the "svgedit/src", you need to use the "built" version (in the distribution folder) -> "svgedit/dist/editor/index.html"

**Reply by michaelliesenberg** [reply]
Hi Jfhenon,

thank you very much, it worked =)
